spring-rabbit/src/test/java/org/springframework/amqp/rabbit/annotation/EnableRabbitIntegrationTests.java

```diff
@@ -1061,6 +1080,30 @@ public String baz(Baz baz, String rk) {
 
 	}
 
+	@Target({ElementType.TYPE, ElementType.METHOD, ElementType.ANNOTATION_TYPE})
+	@Retention(RetentionPolicy.RUNTIME)
+	@RabbitListener(bindings = @QueueBinding(
+			value = @Queue,
+			exchange = @Exchange(value = "test.metaFanout", type = ExchangeTypes.FANOUT, autoDelete = "true")))
+	public @interface MyAnonFanoutListener {
```

Comment:
I wonder if they works well as really full meta-annotation.
I mean how about the `value` attribute in the custom annotation, which is visible from the annotation processor.
Do we use here in the SA the same mechanism as we do in the SI ?
Thanks